src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -37,7 +37,7 @@
             "rtp",
             ForgePermissionNodes.RANDOM_TELEPORT,
             RandomTeleportCommand.class,
-            new ActionArgument("player", ActionArgumentType.PLAYER, ActionArgumentType.Is.OPTIONAL)
+            new ActionArgument("player", ActionArgumentType.ONLINE_PLAYER, ActionArgumentType.Is.OPTIONAL)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "player" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)